ras/ras_ppcdiag.py.data/options.yaml

```diff
@@ -2,3 +2,9 @@ nvsetenv_list: ['load-base', 'load-base 7000']
 usysattn_list: ['-h', '-V', '-P']
 usysfault_list: ['-h', '-V', '-P']
 usysident_list: ['-h', '-V', '-P']
+run_type: !mux
+    upstream:
+        ppcdiag_url: 'https://github.com/power-ras/ppc64-diag/archive/refs/heads/master.zip'
```

Comment:
since we are already hard_coding this in script as default value, we can remove the url here

Reply:
sure, will update